.github/workflows/linux.yml

```diff
@@ -33,8 +33,9 @@ jobs:
       run: sudo apt-get install -y g++-4.8
     - name: Start Xvfb
       run: |
-        Xvfb :99 &
-        disown -ar
-        echo "::set-env name=DISPLAY:::99"
+        sudo apt-get install libwoff1 libopus0 libwebp6 libwebpdemux2 libenchant1c2a libgudev-1.0-0 libsecret-1-0 libhyphen0 libgdk-pixbuf2.0-0
+        sudo apt-get install xvfb
+        export DISPLAY=:99.0
+        Xvfb -ac :99 -screen 0 1280x1024x16 > /dev/null 2>&1 &
     - name: run cargo test
-      run: cargo test --release -- --nocapture
+      run: xvfb-run --auto-servernum cargo test --release -- --nocapture
```

Comment:
Can we remove the entire previous step and use just this? It looks cleaner than forking.

Reply:
This was a great find! It turns out, this was the only line needed and we can remove the entire previous step!

https://github.com/neon-bindings/neon/pull/640